pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -218,6 +218,87 @@ func (m *MigCluster) BuildRestConfig(c k8sclient.Client) (*rest.Config, error) {
 	return restConfig, nil
 }
 
+// GetResources retrieves a list of resources on a cluster
+func (m *MigCluster) GetResources(c k8sclient.Client) ([]*metav1.APIResourceList, error) {
```

Comment:
Mostly already defined in https://github.com/konveyor/mig-controller/blob/master/pkg/gvk/gvk.go#L123